PublishParasoftResults/ParaReportPublishService.ts

```diff
@@ -311,7 +311,14 @@ export class ParaReportPublishService {
     }
 
     transformToCobertura = (sourcePath: string): void => {
-        this.transform(sourcePath, this.COBERTURA_XSL, sourcePath + this.COBERTURA_SUFFIX, this.coberturaReports, true)
+        this.transform(sourcePath, this.COBERTURA_XSL, this.getCoberturaReportFilePath(sourcePath), this.coberturaReports, true)
+    }
+
+    getCoberturaReportFilePath = (sourcePath: string) : string => {
```

Comment:
We should do the same thing for other reports(unit and static analysis), not only just for Cobertura report. I recommend to refactor the metod to 
```
getTransformedReportFilePath = (sourcePath: string, suffix: string) : string => {
    ...
}
```